blazor/treegrid/virtualization.md

```diff
@@ -299,6 +299,114 @@ To setup the column virtualization, set the [EnableVirtualization](https://help.
 The following GIF represent a Tree Grid with Column virtualization.
 ![Column Virtualization in Blazor TreeGrid](./images/blazor-treegrid-column-virtualization.gif)
 
+## Enable Cell placeholder during Virtualization
+
+This enables cell placeholder during virtual scrolling and displays a loading placeholder indicator on the cells while loading new data. Also, the same set of DOM elements is reused to enhance performance.
```

Comment:
This enables the cell placeholder during virtual scrolling and displays a loading placeholder indicator on the cells while loading new data. Also, the same set of DOM elements is reused to enhance performance.